src/main/java/calculator/model/InputValidator.java

```diff
@@ -0,0 +1,74 @@
+package calculator.model;
+
+import java.util.regex.Pattern;
+
+// 입력 검증 클래스
+public class InputValidator {
+
+    // 상수 정의 (기본 구분자, 커스텀 구분자, 개행 문자)
+    private static final String DEFAULT_DELIMITER_REGEX = ",|:";
+    private static final String CUSTOM_DELIMITER_PREFIX = "//";
+    private static final String NEWLINE = "\\n";
```

Comment:
상수화 꼼꼼하게 해주신 것 너무 좋네요!!!
`NEWLINE`은 `"\n"`인데, 지금은 escape 문자를 사용하고 있어서 `NEWLINE`은 아닌 것으로 보이네요.
prefix와 suffix 이름을 맞추면 더 관리하기 쉬워지지 않을까 싶습니다! 👍 
```suggestion
    private static final String CUSTOM_DELIMITER_PREFIX = "//";
    private static final String CUSTOM_DELIMITER_SUFFIX = "\\n";
```